nerdlets/e2m-gui-nerdlet/form-components/rule-description-validator.js

```diff
@@ -1,90 +1,88 @@
-import React from "react";
-import PropTypes from "prop-types";
-import { TextField, Spinner } from "nr1";
+import React from 'react';
+import PropTypes from 'prop-types';
+import { TextField } from 'nr1';
 
 class RuleDescriptionValidator extends React.Component {
-	constructor(props) {
-		super(props);
-		this.state = {
-			ruleDescription: null,
-			validationTimeout: null,
-			validating: false,
-			validated: null
-		};
-	}
+  constructor(props) {
+    super(props);
+    this.state = {
+      ruleDescription: null
+    };
+  }
 
-	clear() {
-		this.setState({
-			ruleDescription: null,
-			validationTimeout: null,
-			validating: false,
-			validated: null
-		});
-	}
+  clear() {
+    this.setState({
+      ruleDescription: null
+    });
+  }
```

Comment:
Obscure?

Reply:
Linting errors brought up that those two fields where unused.